portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,15 +27,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private List<String> names;
+  private List<String> messages;
   @Override
   public void init(){
     names = new ArrayList<>();
     names.add("Obi");
+
+    messages = new ArrayList<>();
+    messages.add("Hello! How are you");
```

Comment:
curious if you can use guava immutablelist.of https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableList.html#of-E-E-E-

Reply:
Ok, I'll try to do a little more research but I'm not sure how I can alter my code to make it fit this.